bip-schnorr.mediawiki

```diff
@@ -147,16 +160,18 @@ All provided signatures are valid with overwhelming probability if and only if t
 ==== Signing ====
 
 Input:
-* The secret key ''d'': an integer in the range ''1..n-1''.
+* The secret key ''d' '': an integer in the range ''1..n-1'' chosen uniformly at random.
```

Comment:
Nit: I don't think you need to mention how the private key is generated in the signing section if you have a separate key generation section.

Reply:
fixed